api/controllers/build.js

```diff
@@ -32,14 +30,26 @@ const emitBuildStatus = build => Site.findByPk(build.site)
     socketIO.to(builderRoom).emit('build status', msg);
     return Promise.resolve();
   })
-  .catch(err => logger.error(err.stack));
+  .catch((err) => {
+    const errBody = {
+      message: 'Failed to emit build status',
+      error: err.stack,
+      buildId: build.id,
+    };
+    EventCreator.error(Event.labels.SOCKET_IO, errBody);
+  });
 
 const saveBuildToProxy = async (build) => {
   const buildSettings = await GithubBuildHelper.fetchContent(build, BUILD_SETTINGS_FILE);
   if (buildSettings) {
     const settings = JSON.parse(buildSettings);
     await ProxyDataSync.saveBuild(build, settings);
-    EventCreator.audit(Event.labels.BUILD_STATUS, build, `${BUILD_SETTINGS_FILE} saved to proxy database`);
+    const body = {
+      filename: BUILD_SETTINGS_FILE,
+      content: settings,
```

Comment:
do we need to store the entire file here? Maybe just the commit sha if you're worried about being able to track this down later.